bbb-common-web/src/main/java/org/bigbluebutton/presentation/imp/SlidesGenerationProgressNotifier.java

```diff
@@ -109,8 +109,6 @@ public void sendConversionCompletedMessage(UploadedPresentation pres) {
       return;
     }
     // Completed conversion -> delete original file
-    pres.deleteOriginalFile();
-
```

Comment:
Maybe you should remove the commentary.
Or change the commentary to explain why it will not remove the file!